system-tests-fixtures/api/complex-api/schema.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
is this schema used yet?

Reply:
@krunalsshah I'm updating this PR.. So a lot will be changed. Please review it again once I push those changes in a bit.